src/api_recievers/influxqueries.js

```diff
@@ -119,7 +118,37 @@ const appData = async (source, appName, currentSession) =>{
     
     await writeClient.close();
   }
+};
+
+const taskData = async(completed, taskName, currentSession) => {
+  if(write === 'true') {
```

Comment:
Don't have to this now but I'm going to at some point delete these write === 'true' statements because I think they just create risk for bugs and there's basically no situation we'd want to not write data at this point